### PR TITLE
feat: adding x-wix-bi-context header to window

### PR DESCRIPTION
**Pull request by kostagersh**
adding sticky x-wix-bi-context header to electron window on open

Closes #WCD-8691

### CODE REVIEW

**Comment by kostagersh** [comment]
I have no idea how to test it. only unit test which isn't helping

**Comment by kostagersh** [comment]
@tombenezra added also to login

File: packages/corvid-cli/src/utils/bi.js [hunk 1]
```diff
@@ -35,8 +35,15 @@ function sendBiEvent(evid) {
   };
 }
 
+function getBiContextHeader(isVisible) {
+  const data = { builderEnv: "local", isHeadless: !isVisible };
+  const encoded = new Buffer(JSON.stringify(data)).toString("base64");
```
Comment (tomenden) [inline packages/corvid-cli/src/utils/bi.js]: 1. When running in `node v10.15` I get this warning:
```events.js:189 (node:4912) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.```
Using - `Buffer.from(JSON.stringify(data)).toString('base64')` works
2. Please verify with @antonpod that this is the expected output

Comment (kostagersh) [reply packages/corvid-cli/src/utils/bi.js]: Done.
Verified with @antonpod. It's ok.

**Pull request merged**